annet/annlib/rbparser/syntax.py

```diff
@@ -78,7 +78,7 @@ def _parse_raw_rule(raw_rule: str, scheme) -> tuple[str, dict[str, str]]:
     row, *params_raw = re.split(r"(?:^|\s)%(?=[a-zA-Z_]\w*)", raw_rule)
     for param in params_raw:
         name, _, value = param.partition("=")
-        params[name] = value.strip() or "1"
+        params[name.strip()] = value.strip() or "1"
 
     row = re.sub(r"\s+", " ", row.strip())
     params = _fill_and_validate(params, scheme, raw_rule)
```

Comment:
This should be fixed as well like this:
```suggestion
    params = _fill_and_validate(params, scheme, row)
```
or like this:
```suggestion
    params = _fill_and_validate(params, scheme, raw_rule.strip())
```


Because `%cant_delete`'s default value is calculated as `raw_rule.startswith("interface")`:
https://github.com/annetutil/annet/blob/main/annet/annlib/rbparser/acl.py#L53
This ^ is the only place in annet that uses callable default value for param.